src/frontend/typer/behandling.ts

```diff
@@ -65,13 +65,14 @@ export enum BehandlingÅrsak {
     ENDRE_MIGRERINGSDATO = 'ENDRE_MIGRERINGSDATO',
     HELMANUELL_MIGRERING = 'HELMANUELL_MIGRERING',
     OMREGNING_SMÅBARNSTILLEGG = 'OMREGNING_SMÅBARNSTILLEGG',
+    BARNEHAGELISTE = 'BARNEHAGELISTE',
 }
 
 export const behandlingÅrsak: Record<BehandlingÅrsak | TilbakekrevingsbehandlingÅrsak, string> = {
     SØKNAD: 'Søknad',
     FØDSELSHENDELSE: 'Fødselshendelse',
     ÅRLIG_KONTROLL: 'Årlig kontroll',
-    DØDSFALL_BRUKER: 'Dødsfall bruker',
+    DØDSFALL_BRUKER: 'Dødsfall',
```

Comment:
```suggestion
    DØDSFALL: 'Dødsfall',
```

Det bør forsåvidt være 1:1 med enum'n fra backend, så jeg tenker at vi kan rename den til DØDSFALL begge steder. Lager en huskelapp på at vi gjør de samme endringene i backend når vi starter der 😊